tasks/s3roundtrip.py

```diff
@@ -28,13 +28,17 @@ def download(ctx, config):
     The context passed in should be identical to the context
     passed in to the main task.
     """
-    assert isinstance(config, list)
+    assert isinstance(config, dict)
```

Comment:
Upon further review, I believe that the finally: section is correct.  I have tested this with force-branch ceph-master, force-branch aardvark, and force-branch hammer and all three cases seem to behave as expect (aardvark is an error).